src/set_theory/game/birthday.lean

```diff
@@ -101,13 +101,16 @@ using_well_founded { dec_tac := pgame_wf_tac }
 by rw [birthday_def, max_eq_zero, lsub_eq_zero_iff, lsub_eq_zero_iff]
 
 @[simp] theorem birthday_zero : birthday 0 = 0 :=
-by { rw birthday_eq_zero, split; apply_instance }
+by simp [pempty.is_empty]
 
 @[simp] theorem birthday_one : birthday 1 = 1 :=
-begin
-  have : (λ i, (move_left 1 i).birthday) = λ i, 0 := funext (λ x, by simp),
-  rw [birthday_def, @lsub_empty (right_moves 1), this, lsub_const, succ_zero, max_zero_right]
-end
+by { rw birthday_def, simp }
+
+@[simp] theorem birthday_star : birthday star = 1 :=
+by { rw birthday_def, simp }
+
+@[simp] theorem birthday_half : birthday half = 2 :=
+by { rw birthday_def, simp, exact max_eq_right (lt_succ_self 1).le }
```

Comment:
This is a non-terminal simp. Can you change it to either a `simpa` or `simp only`?

Reply:
`simpa using max_eq_right (lt_succ_self 1).le` won't work for whatever reason. I can make it a `simp only`, even though it ends up taking much more space.

Reply:
Does `simp [max_eq_right (lt_succ_self 1)]` work?

Reply:
Also no. I think the problem might be that `simp` can't see through the def-eq `succ 1 = 2`.

Reply:
We have the lemma `fin.succ_one_eq_two`; I don't think it would be unreasonable to have it here too.

Reply:
`simp [(show 2 = (1 : ordinal).succ, from rfl), (lt_succ_self 1).le],` also works

Reply:
Adding this `succ_one` lemma makes `simpa` work.